x-pack/test/case_api_integration/security_and_spaces/tests/trial/cases/push_case.ts

```diff
@@ -95,6 +102,54 @@ export default ({ getService }: FtrProviderContext): void => {
       ).to.equal(true);
     });
 
+    it('should create the mappings when pushing a case', async () => {
+      // create a connector but not a configuration so that the mapping will not be present
+      const connector = await createConnector({
+        supertest,
+        req: {
+          ...getServiceNowConnector(),
+          config: { apiUrl: servicenowSimulatorURL },
+        },
+      });
+
+      const postedCase = await createCase(
+        supertest,
+        {
+          ...getPostCaseRequest(),
+          connector: {
+            id: connector.id,
+            name: connector.name,
+            type: connector.connector_type_id,
+            fields: {
+              urgency: '2',
+              impact: '2',
+              severity: '2',
+              category: 'software',
+              subcategory: 'os',
+            },
+          } as CaseConnector,
+        },
+        200
+      );
+
+      // there should be no mappings initially
+      let mappings = await getConnectorMappingsFromES({ es });
+      expect(mappings.body.hits.hits.length).to.eql(0);
+
+      await pushCase({
+        supertest,
+        caseId: postedCase.id,
+        connectorId: connector.id,
+      });
+
+      // the mappings should now be created after the push
+      mappings = await getConnectorMappingsFromES({ es });
+      expect(mappings.body.hits.hits.length).to.be(1);
+      expect(
+        mappings.body.hits.hits[0]._source?.['cases-connector-mappings'].mappings.length
+      ).to.be.above(0);
```

Comment:
@cnasikas Not sure if it makes sense to check the actual mappings themselves? Should there always be 3 in the array?

Reply:
At the moment, yes only 3 items will be on the array. I think is fine to check only for existence (greater than zero). We have [other integration tests](https://github.com/elastic/kibana/blob/master/x-pack/test/case_api_integration/security_and_spaces/tests/common/configure/post_configure.ts#L76) that checks each connector's mapping.